notify/share/python/glpi.py

```diff
@@ -0,0 +1,147 @@
+import requests, json
+
+# TODO clean initalizing to create the adequate ticket
+
+# Class used to create the php filter to look for objects in the glpi api
+class criterias:
+  def __init__(self):
+    self.array = []
+
+  def addCriteria(self, criteria):
+    self.array.append(criteria)
+
+  def __str__(self):
+    output = ""
+    for iCriteria in range(len(self.array)):
+      output += (self.array[iCriteria]).show(iCriteria) + "&"
+    return output[:len(output)-1]
+      
+class criteria:
+  def __init__(self, field, searchType, value, link="AND"):
+    self.link = link
+    self.field = field
+    self.searchtype = searchType
+    self.value = value
+
+  def show(self, i):
+    sep = "criteria[" + str(i) + "]"
+    return sep + "[link]=" + self.link + "&" + sep + "[field]=" + self.field + "&" + sep + "[searchtype]=" + self.searchtype + "&" + sep + "[value]=" + self.value
+    
+
+
+# Class used to create a ticket in the glpi api
+class glpiTicket:
+  def __init__(self, **fields): 
+    self.__dict__ = fields
+
+  def __init__(self, name, content, status=1, urgency=3, impact=3, priority=3):
+    self.name = name
+    self.content = content
+    self.status = status
+    self.urgency = urgency
+    self.impact = impact
+    self.priority = priority
+#    self.entities_id = entities_id
```

Comment:
why are the commented.

Reply:
They are not used here, but may be in future updates